spec/factories/models.rb

```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
```

Comment:
@katerspotaters this needs to be removed, please. Just commit to the same branch.